mate-settings-daemon/Makefile.am

```diff
@@ -41,23 +41,24 @@ msddir = $(libexecdir)
 msd_PROGRAMS = \
 	mate-settings-daemon
 
-mate-settings-manager-glue.h: mate-settings-manager.xml Makefile.am
-	$(AM_V_GEN) dbus-binding-tool --prefix=mate_settings_manager --mode=glib-server $< > $@
-
-mate-settings-client.h: mate-settings-manager.xml Makefile.am
-	$(AM_V_GEN) dbus-binding-tool --prefix=mate_settings_manager --mode=glib-client $< > $@
-
 BUILT_SOURCES = 			\
-	mate-settings-manager-glue.h	\
-	mate-settings-client.h		\
+	manager-dbus-generated.h \
+	manager-dbus-generated.c \
 	$(NULL)
 
+$(BUILT_SOURCES) : Makefile.am $(srcdir)/mate-settings-manager.xml
+	$(AM_V_GEN) gdbus-codegen \
```

Comment:
You can use `$(GDBUS_CODEGEN)` instead of  `gdbus-codegen`
```
<cut>
        $(AM_V_GEN) $(GDBUS_CODEGEN) \
<cut>
```
```patch
diff --git a/configure.ac b/configure.ac
index ac38ed5..1ecdd5e 100644
--- a/configure.ac
+++ b/configure.ac
@@ -83,6 +83,7 @@ AC_SUBST([MSD_PLUGIN_LDFLAGS])
 PKG_CHECK_MODULES(DCONF, dconf >= 0.13.4)
 
 AC_PATH_PROG(GLIB_GENMARSHAL, glib-genmarshal)
+AC_PATH_PROG(GDBUS_CODEGEN, gdbus-codegen)
 
 GLIB_GSETTINGS
 
```